### DISCUSSION TITLE
Websockets

### DISCUSSION THREAD

**Discussion by ghost** [matt-42/lithium#55] [Ideas]
Do you plan on adding websocket handling?
I really like the library so far, but I require websockets for additional communication in a personal project.
(It will be some kind of real time physics multiplayer game if the idea works.)

**Comment by matt-42** [comment]
This could be a nice addition for one of my next lithium rush. But do not wait for it anytime soon, I'm very busy on my current job right now. Maybe I'll have a prototype at the end of this year...
